devel/xdispatch2/Portfile

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           codeberg 1.0
+PortGroup           conflicts_build 1.0
+
+codeberg.setup      emzeat xdispatch2 2.1.3 v
+revision            0
+categories          devel
+license             Apache-2
+maintainers         {@barracuda156 gmail.com:vital.had}
```

Comment:
what is special about this port that it cannot be `openmaintainer`?

Reply:
It can. (It is perhaps more important for older systems – or non-MacOS ones, but I don’t mind at all.)